translations/base-fr.yaml

```diff
@@ -794,6 +869,21 @@ settings:
             description: >-
                 Le jeu est divisé en secteurs de 16×16 cases. Si ce réglage est activé, les limites de chaque secteur sont affichées.
 
+        pickMinerOnPatch:
+            title: Choisir un extracteur sur un emplacement de ressource.
+            description: >-
+                Activé par défaut, sélectionne l'extracteur si vous utilisez la pipette lors du survol d'un emplacement de ressource.
+
+        simplifiedBelts:
+            title: Convoyeurs simplifiés (Laid)
+            description: >-
+                Désactive le rendu des éléments des convoyeurs sauf lors du survol du convoyeurs pour économiser des performances. Je ne recommande pas de jouer avec ce réglage si vous n'avez pas absolument besoin de performance.
```

Comment:
typo: `du convoyeurs`. Singular and plural both sound fine to me, but not both at once

Reply:
Oops!